HW32/src/main/java/com/example/webapp/root/repository/OrderService.java

```diff
@@ -0,0 +1,28 @@
+package com.example.webapp.root.repository;
+
+import com.example.webapp.root.entity.Order;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Optional;
+
+@Service
+@RequiredArgsConstructor
+public class OrderService {
+
+    private final OrderRepository orderRepository;
```

Comment:
It is better to use an explicit declaration of a component, like `@Autowired` or in a constructor. Annotation from `Lombok` lib is not the best to declare a component.